src/game/overlay.c

```diff
@@ -78,12 +80,90 @@ static RGB888 m_ColorBarMap[][COLOR_STEPS] = {
       { 120, 60, 70 } },
 };
 
+typedef struct BAR_INFO {
+    int32_t type;
+    int32_t value;
+    int32_t max_value;
+    int32_t show;
+    int32_t show_mode;
+    int32_t blink;
+    int32_t timer;
+    int32_t color;
+    int32_t bar_location;
+} BAR_INFO;
+
+static BAR_INFO m_HealthBar = { 0 };
+static void Overlay_SetupHealthBar()
+{
+    m_HealthBar.type = BT_LARA_HEALTH;
+    m_HealthBar.value = LARA_HITPOINTS;
+    m_HealthBar.max_value = LARA_HITPOINTS;
+    m_HealthBar.show_mode = g_Config.healthbar_showing_mode;
+    m_HealthBar.show = 0;
+    m_HealthBar.blink = 0;
+    m_HealthBar.timer = 40;
+    m_HealthBar.color = g_Config.healthbar_color;
+    m_HealthBar.bar_location = g_Config.healthbar_location;
+}
+
+static BAR_INFO m_AirBar = { 0 };
```

Comment:
Let's lump all static variables together, then all static functions, then all public functions.
Also all functions should have a forward declaration (I know they may not be needed, but this is for consistency so that we don't have to think about it.)